doc/source/tune/api/trainable.rst

```diff
@@ -248,4 +248,5 @@ Tune Trainable Debugging Utilities
 
     tune.utils.diagnose_serialization
     tune.utils.validate_save_restore
-
+    tune.utils.util.validate_warmstart
+    tune.utils.log.Verbosity
```

Comment:
This is replaced by `AirVerbosity`, but it's also not officially deprecated 😞 . Can you remove + whitelist this for now and add a deprecation `TODO(ml-team)`?

The whole "tune output" module needs to be cleaned up since it's in an in-between state at the moment.

Reply:
totally

Reply:
oh can we remove the `Verbosity` from this list then? and add it to the whitelist?

Reply:
w00t yes good catch